src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -120,9 +120,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        // Create the bundles directory otherwise symlink will fail.
         $bundlesDir = $targetArg.'/bundles/';
-        $this->filesystem->mkdir($bundlesDir, 0777);
```

Comment:
given symlink() creates the parent directory, and mkdir() handles parents recursively, we could drop this line.